client-participation/js/strings/en.js

```diff
@@ -0,0 +1,114 @@
+  "notSentSinceDemo": "(not really, this is a demo)",
+  "submitComment": "Submit",
+  "tipStarred": "Marked as important.",
+  "participantHelpWelcomeText": "Welcome to a new kind of conversation - <em>vote</em> on other people's statements.",
+  "participantHelpGroupsText": "People who vote similarly <span style='font-weight: 700;'>are grouped.</span> Click a group to see which viewpoints they share. <a style='font-weight: 700; cursor: pointer; text-decoration: underline' id='helpTextGroupsExpand'>...more</a>",
+  "participantHelpGroupsNotYetText": "The visualization will appear once 7 participants have begun voting",
+  "helpWhatAreGroupsDetail": "<p>You've probably seen 'recommended products' on Amazon, or 'recommended movies' on Netflix. Each of those services uses statistics to group the user with people who buy and watch similar things, then show them things that those people bought or watched.</p> <p> When a user votes on statements, they are grouped with people who voted like they did! You can see those groups below. Each is made up of people who have similar opinions. There are fascinating insights to discover in each conversation. Go ahead - click a group to see what brought them together and what makes them unique! </p>",
+  "socialConnectPrompt": "Optionally connect to see friends and people you follow in the visualization.",
+  "connectFbButton": "Connect with Facebook",
+  "connectTwButton": "Connect with Twitter",
+  "polis_err_reg_fb_verification_email_sent": "Please check your email for a verification link, then return here to continue.",
+  "polis_err_reg_fb_verification_noemail_unverified": "Your Facebook account is unverified. Please verify your email address with Facebook, then return here to continue.",
+  "showTranslationButton": "Activate third-party translation",
+  "hideTranslationButton": "Deactivate Translation",
+  "thirdPartyTranslationDisclaimer": "Translation provided by a third party",
+
+  "notificationsAlreadySubscribed": "You are subscribed to updates for this conversation.",
+  "notificationsGetNotified": "Get notified when more statements arrive:",
+  "notificationsEnterEmail": "Enter your email address to get notified when more statements arrive:",
+  "labelEmail": "Email",
+  "notificationsSubscribeButton": "Subscribe",
+  "notificationsSubscribeErrorAlert": "Error subscribing",
+  "noCommentsYet": "There aren't any statements yet.",
+  "noCommentsYetSoWrite": "Get this conversation started by adding a statement.",
+  "noCommentsYetSoInvite": "Get this conversation started by inviting more participants, or add a statement.",
+  "noCommentsYouVotedOnAll": "You've voted on all the statements.",
+  "noCommentsTryWritingOne": "If you have something to add, try writing your own statement.",
+  "convIsClosed": "This conversation is closed.",
+  "noMoreVotingAllowed": "No further voting is allowed.",
+
+  "demo": "This is a demo string!"
```

Comment:
Remove before merging (should also remove string from Mojito db via `push-translations` workflow):

```suggestion
```